ETF quickstart documentation

jenriquesoriano opened the pull request:
ETF quickstart documentation for OSGeoLive 15

astroidex commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,258 @@
+Finally the test can be started by clicking on the Start button. The view then changes automatically to the Monitor View.
+
+Monitor test runs
+============
```
=== same length as text above

jenriquesoriano replied:
same length for the ======

astroidex commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,258 @@
+.. image:: /images/projects/ETF/test-reports-1.png
+    :scale: 70 %
+
+By clicking on the plus button information, about the start time, the test result status, the name of the Test Object and the used Test Suites is shown.
```
are shown ?

jenriquesoriano replied:
true, corrected

astroidex commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,258 @@
+
+
+Inspect test reports
+============
```
length ====

jenriquesoriano replied:
same length for the ======

astroidex commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,258 @@
+.. image:: /images/projects/ETF/inspect-test-reports-6.png
+    :scale: 70 %
+
+
```
The quickstarts normally end with a What's next? caption. Maybe you can add something like that
https://github.com/OSGeo/OSGeoLive-doc/blob/master/doc/quickstart/R_quickstart.rst

jenriquesoriano replied:
Resources and Whats next added

astroidex reviewed (changes requested):
Only some comments in the quickstart. Are you working on the overview too?

jenriquesoriano commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,258 @@
+
+This Quick Start describes how to:
+
+  * navigate through the web applicattion
```
Typo corrected

jenriquesoriano commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,258 @@
+.. image:: /images/projects/ETF/introduction.png
+    :scale: 70 %
+    
+#. First one is Start test. In this section all available (respectively installed) Executable Test Suites are listed. Within this section, an Executable Test Suite can be selected and run against a Test Object..
```
done, thanks. And some additional text

jenriquesoriano commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,258 @@
+    
+#. First one is Start test. In this section all available (respectively installed) Executable Test Suites are listed. Within this section, an Executable Test Suite can be selected and run against a Test Object..
+
+#. Second one is Status. This one shows all tests that are currently executed on the system and enables to open a monitor view for single test runs. can check the status of any running test. Moreover, below the running tests appear the components currently loaded
```
rephrased, thanks!

jenriquesoriano commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,258 @@
+
+#. Second one is Status. This one shows all tests that are currently executed on the system and enables to open a monitor view for single test runs. can check the status of any running test. Moreover, below the running tests appear the components currently loaded
+
+#. Third one is Test reports. In this one the results any finished test can be checked, opened to see more details or downloaded.
```
rephrased

jenriquesoriano commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,258 @@
+Dependencies and Parameters
+----------------------------------
+
+The Test Suites button shows some basic information about the selected Test Suites and -if applicable- about the direct dependencies.
```
true, done (and rephrased)

jenriquesoriano commented:
Dear @astroidex , thanks a lot for your revision.
We have addressed all your comments, also adding some additional input.
We think it is ready for your revision.
Please let us know if further input is needed.

PS: we are incorporating the overview in a separate PR.

jenriquesoriano commented:
Included ETF line in projects_info.csv under "Browser Facing GIS->Others".

cvvergara commented:
@jenriquesoriano 
I can see that you have also a branch for the [overview](https://github.com/etf-validator/OSGeoLive-doc/tree/etf-overview)
Can you add those changes to this branch.
So that everything gets processed.

Also some images are missing,
This would be the equivalent of the missing images:
https://github.com/OSGeo/OSGeoLive-doc/tree/master/doc/images/projects/pgrouting
The names of the images have to be:
logo_ETF.png
ETF_screenshot.png
There are scripts that reference those files to build, for example, the [presentation](https://osgeo.github.io/OSGeoLive-doc/en/presentation.html#/)


Thanks.

cvvergara commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,286 @@
+
+
+.. |GS| replace:: GeoServer
+.. @NAME_ETF@ replace::ETF
```
This is not needed, it is done automatically with our scripts

cvvergara commented on doc/quickstart/ETF_quickstart.rst:
```diff
@@ -0,0 +1,286 @@
+@NAME_ETF@ Quickstart
+********************************************************************************
+
+ETF is an open source testing framework for validating spatial data, metadata and web services in Spatial Data Infrastructures (SDIs). The design of ETF is driven by three goals: be user-friendly, consistent with the standards and capable of testing all resources in an SDI.
```
Not urgent:
There are lines that are way over 120 characters. (kinda these three lines is one line)
More or less this is how I see the file:
https://raw.githubusercontent.com/OSGeo/OSGeoLive-doc/4209588cf4a43cd148e07941e162d633ef09fb5d/doc/quickstart/ETF_quickstart.rst
(Maybe your editor gives you the impression that is changing lines)

When there are little issues remaining for me to fix (issues are because of links or sphinx formatting) then
I re-format those long lines. and make them 80 chars long.
No issues, I don't touch a thing.

I reiterate, its not urgent, more urgent is to have the overview on the same PR.
The last thing I worry about is formatting. but is worth to mention.

cvvergara commented:
@jenriquesoriano 
The overview is missing:
```
https://github.com/OSGeo/OSGeoLive-doc/runs/7266725376?check_suite_focus=true#step:6:20
```

cvvergara commented:
@jenriquesoriano 
The quickstart and overview have been merged in #748 where all thechnical problems are solved.

For the moment, EMT is located on [Spatial tools](https://osgeo.github.io/OSGeoLive-doc/en/overview/overview.html#spatial-tools)

Please double check the overview and the quickstart, and if needed, do a PR.

jenriquesoriano commented:
Dear @cvvergara  , 

thanks a lot for merging and accepting the PR. 
Indeed we are working to provide a better version of the overview, more aligned with the rest of the tools, but for us it is perfectly OK to create a new PR accordingly.
Thanks again for all your efforts.

Best
